typescript/src/async-rust-call.ts

```diff
@@ -63,6 +63,7 @@ export async function uniffiRustCallAsync<F, T>(
 
   // We now poll the Rust future until it's ready.
   // The poll, complete and free methods are specialized by the FFIType of the return value.
+  let _stack = new Error().stack;
```

Comment:
🥴 This would be done on every async call.

Is generating the stack trace of the same order of magnitude of performance of the FFI?

Is it worth it?

Reply:
In a debug build, probably worth it, or at least worth trying maybe?  In a release build, probably not?

I think we could guard this with a `process.env.NODE_ENV !== 'production'` conditional